Assets/MRTK/Examples/Experimental/SceneUnderstanding/Scripts/DemoSceneUnderstandingController.cs

```diff
@@ -426,6 +427,17 @@ private Color ColorForSurfaceType(SpatialAwarenessSurfaceTypes surfaceType)
             }
         }
 
+        private void ClearAndUpdateObserver()
+        {
+            foreach (GameObject gameObject in instantiatedPrefabs)
+            {
+                Destroy(gameObject);
+            }
+            instantiatedPrefabs.Clear();
+            observer.ClearObservations();
```

Comment:
This block looks the same as the above ClearScene method. Can we reuse?

![image](https://user-images.githubusercontent.com/3580640/107294464-48ef3280-6a22-11eb-8e5e-d5a497219257.png)